test/common/data.js

```diff
@@ -0,0 +1,22 @@
+const { Vendor, Carrier, Pharmacy } = require('./participantCategories');
+const carrierCategories = require('./carrierCategories');
+
+const randomTemperature = () => Math.floor(-20 * Math.random());
+
+const participant = (id, category) => ({
+  id,
+  category,
+});
+
+const conditions = (category = carrierCategories.Truck) => ({
+  temperature: randomTemperature(),
+  category,
+});
+
+module.exports = {
+  vendor: id => Object.assign({}, participant(id, Vendor)),
+  carrier: (id, carrierCategory) => Object.assign(
+    {}, participant(id, Carrier), { conditions: conditions(carrierCategory) },
+  ),
+  pharmacy: id => Object.assign({}, participant(id, Pharmacy)),
```

Comment:
since node 10 you can use spread object syntax e.g. ` pharmacy: id => { ...participant(id, Pharmacy)}`